Some needed updates to remove broken links and out of date info

prushforth opened the pull request:
- Remove cubewerx examples
- Remove link to mapml-client library
- Add links to explainer and meta-explainer blog post
- Replace link to MapML Servlet with links to GeoServer MapML Community Module

prushforth committed:
Remove CubeWerx examples, as they have stopped serving MapML at this time.
Add vector example (which is still based on the polymer shim and consequently
shows a FOUC.  I will update that example sometime soon to use the web-map
custom element release.

prushforth committed:
Add favicon.ico from w3c wordpress.

Malvoz commented on index.html:
```diff
@@ -197,26 +210,21 @@ <h2>Software projects</h2>
           </p>
         </li>
-        <li><a href="https://github.com/Maps4HTML/MapMLServer">MapML Server</a>
+        <li><a href="https://github.com/Maps4HTML/geoserver">GeoServer MapML MapML Community Module</a>
```
Should this say "GeoServer MapML MapML Community Module"?

prushforth replied:
Lol no, thanks, I'll fix it Thanks for your review.

prushforth committed:
Correct typo

prushforth committed:
Add validator, parser to software links

Malvoz commented:
Would [RFC 5870](https://tools.ietf.org/html/rfc5870) be a good addition to the "Other links" section? (we've already got an issue in the Use Cases & Requirements repo to discuss use cases for it)

prushforth commented:
> Would [RFC 5870](https://tools.ietf.org/html/rfc5870) be a good addition to the "Other links" section? (we've already got an issue in the Use Cases & Requirements repo to discuss use cases for it)

Sure thing, will add a commit in the morning.

prushforth committed:
Add RFC standards section, plus other useful stuff

prushforth committed:
Update .gitignore for Peter's invasive IDE files

AmeliaBR commented on index.html:
```diff
@@ -235,14 +255,25 @@ <h2>Working Demos</h2>
+        </li>
+
+        
+<!--        <li><a href="https://geogratis.gc.ca/mapml/static/cubewerx_cgdi_proxy_geomet_directory.html">MapML versions of GeoMet WMS</a>
```
Instead of commenting this out, why not directly remove it?

prushforth replied:
I was hoping that Cubewerx would consider my request to restore their services. But yes, I will delete it. We can always recover from the commit history.

AmeliaBR commented on index.html:
```diff
@@ -235,14 +255,25 @@ <h2>Working Demos</h2>
         <li><a href="https://geogratis.gc.ca/mapml/static/cubewerx_cgdi_proxy_wmts.html">Collection of MapML viewers of CubeWerx maps</a>
+        </li>-->
+        <li><a href="https://www.youtube.com/channel/UCKIv6IGwzxPF2HNhvq-VhKA" target="_blank">Maps for HTML is on YouTube</a>
+          <p>The Maps for HTML Community Group has a <a href="https://www.youtube.com/channel/UCKIv6IGwzxPF2HNhvq-VhKA" target="_blank">channel</a> on YouTube.</p>
```
This doesn't really belong in the “Working Demos” section.

Maybe add to the “About the Community Group” section? We should also add the Twitter account there, too.

Also, `target="_blank"` is unnecessary. This is a static page, the browser back button works fine if someone needs to get back to the list!

prushforth replied:
OK

AmeliaBR commented on index.html:
```diff
@@ -303,6 +334,27 @@ <h2>Other links</h2>
       </ul>
     </section>
+    <section>
+      <h2>RFCs and other relevant documents</h2>
```
Maybe make this “Related standards and document formats” and also include the OGC link? (Or call out specific OGC standards even)  And probably move it up before the general “Other Links” section?

prushforth replied:
I put the OGC standards in the renamed section as well.

AmeliaBR commented on index.html:
```diff
@@ -257,16 +288,18 @@ <h2>Other links</h2>
           </p>
         </li>
-        <li><a href="https://docs.opengeospatial.org/per/17-019.html">MapML Engineering Report (2017)</a> from the <abbr title="Open Geospatial Consortium">OGC</abbr> Testbed 13
+        <li>MapML Engineering Reports from the <abbr title="Open Geospatial Consortium">OGC</abbr> Innovation Program
```
Don't need the `target="_blank"` for the links.

Also: Maybe move these all up to the end of the “Specifications and other reports” section, so they're not lost in “Other Links”?

prushforth replied:
OK

AmeliaBR reviewed (commented):
Lots of great additions here. Makes me reconsider the overall organization though: see comments.

prushforth committed:
Merge branch 'master' of https://github.com/Maps4HTML/Maps4HTML.github.io

prushforth committed:
Updates requested during review.  Also moved OGC standards section into
the Related standards section (which was renamed, so now can logically
be the parent of this content).
